docker-compose.yml

```diff
@@ -0,0 +1,98 @@
+version: '3.7'
+
+volumes:
+    grafana_data: {}
+    prometheus_data: {}
+
+networks:
+  yearn-exporter:
+
+services:
+  yearn-exporter-v1:
+    build: .
+    command: exporter_v1
+    environment:
+      - WEB3_INFURA_PROJECT_ID
+      - ALCHEMY_URL
+      - ETHERSCAN_TOKEN
+    networks:
+      - yearn-exporter
+    restart: always
+
+  yearn-exporter-v2:
+    build: .
+    command: exporter_v2
+    environment:
+      - WEB3_INFURA_PROJECT_ID
+      - ALCHEMY_URL
+      - ETHERSCAN_TOKEN
+    networks:
+      - yearn-exporter
+    restart: always
+
+  yearn-exporter-iearn:
+    build: .
+    command: exporter_iearn
+    environment:
+      - WEB3_INFURA_PROJECT_ID
+      - ALCHEMY_URL
+      - ETHERSCAN_TOKEN
+    networks:
+      - yearn-exporter
+    restart: always
+
+  yearn-exporter-ironbank:
+    build: .
+    command: exporter_ironbank
+    environment:
+      - WEB3_INFURA_PROJECT_ID
+      - ALCHEMY_URL
+      - ETHERSCAN_TOKEN
+    networks:
+      - yearn-exporter
+    restart: always
+
+  yearn-exporter-experimental:
+    build: .
+    command: exporter_experimental
+    environment:
+      - WEB3_INFURA_PROJECT_ID
+      - ALCHEMY_URL
+      - ETHERSCAN_TOKEN
+    networks:
+      - yearn-exporter
+    restart: always
+
+  prometheus:
+    image: prom/prometheus:v2.23.0
+    volumes:
+      - ./prometheus/:/etc/prometheus/
+      - prometheus_data:/prometheus
+    command:
```

Comment:
Prometheus deletes data after two weeks by default, add `--storage.tsdb.retention.time 1y`

Reply:
haha that's a good catch, this would have been funny after two weeks :facepalm: